src/tools/pontoon/adding_new_locale.md

```diff
@@ -101,4 +101,4 @@ Population: 1863468
 
 Find the locale code in [CLDR territoryInfo.json](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) and multiply its `_populationPercent` with the territory `_population`. Repeat if multiple occurrences of locale code exist and sum products.
 
-At this point click **SAVE** in the bottom right corner to save and create the new locale. Then, from the [standard admin interface](https://pontoon.mozilla.org/admin/), enable at least one “non system project” (i.e. not `Tutorial` or `Pontoon Intro`) and wait until Pontoon syncs for the locale dashboard to work. You can then check if the locale’s page, as well as the new project, are available - in this case at https://pontoon.mozilla.org/am/
+At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should then appear under the new team's page. Wait until Pontoon syncs for the locale dashboard to work. You can then check if the locale’s page, as well as the new project, are available - in this case at https://pontoon.mozilla.org/am/
```

Comment:
```suggestion
At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should then appear under the new team's page. You can then check if the locale’s page is available - in this case at https://pontoon.mozilla.org/am/
```

At this point, we don't even need to wait for the sync.

Reply:
Done!